src/main/groovy/org/transmartproject/batch/concept/InsertConceptsTasklet.groovy

```diff
@@ -22,6 +22,7 @@ import org.transmartproject.batch.support.SecureObjectToken
 @Component
 @JobScopeInterfaced
 @Slf4j
+//TODO Reuse @{see ConceptsTablesWriter}
 class InsertConceptsTasklet implements Tasklet {
```

Comment:
Yes, this smells. The classes are very much alike. Seems like the business logic could could easily be moved to a job-scoped service bean in the `batch.concept` package that neither implements `Tasklet` nor `ItemWrite` that would be called by `InsertConceptsTasklet` and `ConceptsTablesWriter`.